ansible/roles/ami-ovf/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+    - name: Disable SELinux
+      selinux:
+        policy: targeted
+        state: permissive
+
+    - name: PMM                        | Add firewalld rule
+      firewalld: port={{ item }} permanent=true state=enabled immediate=yes
+      with_items:
+          - 80/tcp
+          - 443/tcp
+
+    - name: PMM                        | Add script which show PMM URL
+      copy:
+        src: show-pmm-url
+        dest: /var/lib/cloud/scripts/per-boot/show-pmm-url
+        mode: 0755
+
+    - name: PMM                        | Find the main js file
+      find:
+        paths: "/usr/share/pmm-server/password-page"
+        patterns: "main.*.js"
+      register: jsresult
```

Comment:
We should remove everything related to password page.

Reply:
ok